### DISCUSSION TITLE
How to get the featured chart in Budibase's landing page

### DISCUSSION THREAD

**Discussion by daawaan4x** [Budibase/budibase#5345] [Help]
Hello, I've been wondering how is this possible? It looks very neat and I can't mimic it with the donut chart

![image](https://user-images.githubusercontent.com/53972232/162694715-68c91e18-f11a-434f-bcfc-72db618e869a.png)

**Comment by joebudi** [comment]
Hi Daawan

This is not currently possible, and the chart above is for marketing purposes. Charts are an area we would like to improve in the future so any feedback is welcomed!

**Reply by daawaan4x** [reply]
Hello @joebudi, that's unfortunate to hear. By the way, I've recently started contributing in Budibase with #5342 and is there a discord server in Budibase with contributors where design specs are shared? I've seen a couple of google docs links in some issues but unfortunately, they're not publicly accessible, and I'd be glad if I can interact together with other contributors and the official team
